packages/web-app/src/lib/get-address.js

```diff
@@ -0,0 +1,14 @@
+const getAddress = (siteAddress) =>
+  [
+    siteAddress?.addressLine1,
+    siteAddress?.addressLine2,
+    siteAddress?.town,
+    siteAddress?.county,
+    siteAddress?.postcode,
+  ].filter((n) => n);
+
+module.exports = {
+  getAddress,
+  getAddressSingleLine: (siteAddress) => getAddress(siteAddress).join(', '),
+  getAddressMultiLine: (siteAddress) => getAddress(siteAddress).join('\n'),
+};
```

Comment:
It appears that these methods are only used in the unit tests? If so, I think a better location for the file would be `test/utils`.

Reply:
i removed getAddress. getAddressSingleLine and getAddressMultiLine are used in notify.js

Reply:
:+1: